front/lib/api/tables.ts

```diff
@@ -369,14 +369,14 @@ export async function rowsFromCsv({
 
   let i = 0;
   const parser = parse(csv, { delimiter });
-  const valuesByCol: Record<string, string[]> = {};
+  const valuesByCol: Record<string, string[]> = Object.create(null);
```

Comment:
Can you explain a bit more? :)

Reply:
mb completely forgot

Reply:
here it was `{}["constructor"]` that caused the issue

Reply:
https://app.datadoghq.eu/logs?query=%22Error%20uploading%20table%20to%20Dust.%22&agg_m=count&agg_m_source=base&agg_t=count&cols=host%2Cservice&fromUser=true&messageDisplay=inline&refresh_mode=paused&storage=hot&stream_sort=desc&viz=stream&from_ts=1733319867925&to_ts=1733319925679&live=false

Reply:
:+1: